Assets/Plugins/Linux/Editor/PlatformSpecificConfigEditorLinux.cs

```diff
@@ -27,34 +27,11 @@
 
 namespace PlayEveryWare.EpicOnlineServices
 {
-    public class PlatformSpecificConfigEditorLinux : IPlatformSpecificConfigEditor
+    public class ConfigSectionLinux : ConfigSection<EOSLinuxConfig>
```

Comment:
This is an artifact of renaming the interface `IPlatformSpecificConfigEditor` to ultimately be an abstract class `ConfigSection` because ReSharper wanted to be helpful and rename more than I asked it to. Consequently, the filename no longer matches the class name - so this should be fixed and is reason enough to request changes.